app/helpers/application_helper.rb

```diff
@@ -36,4 +36,8 @@ def service_link
   def current_namespace
     params[:controller].split('/').first
   end
+
+  def full_name(application_form)
```

Comment:
This probably warrants to be put in `ApplicationForm`.

Reply:
it sounds good (possible we'll need a Container for that view, so I might put full_name in it)